jupyter_elfinder/elfinder.py

```diff
@@ -1113,10 +1126,10 @@ def __info(self, path: str) -> Info:
                         tmb_url = self.__path2url(tmb)
                         info["tmb"] = tmb_url
                     else:
-                        # FIXME: This is not part of api 2.1
-                        self._response[R_TMB] = True
                         if info["mime"].startswith("image/"):
-                            info["tmb"] = 1
+                            info["tmb"] = "1"
+                        else:
+                            info["tmb"] = None
```

Comment:
I'm wondering if we should not set it at all in this case, instead of setting it to `None`.

Reply:
That would also work, no strong opinion. In JS, None will be `null`, if we don't set, it will be `undefined`. Both `null` and `undefined` will be treated as false.

Reply:
For the typing purpose, I think it's better to be explicit though. `"tmb": Union[str, None]` would mean this one is optional.

Reply:
No, the info TypedDict is already set `total` to `False` which means it doesn't need to be complete. That's also the intention of the api as I read it. We don't need to set all keys in the info object.

It's a lot better to have stringent typing than unions or optionals.